src/pages/WeatherForecast/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { FC, useContext, useEffect, useState, SyntheticEvent } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Typography, Tabs, Button, Container } from '@mui/material';
+
+import { isEmpty } from 'helpers/isEmpty';
+import { api, IWeatherItem } from 'helpers/api';
+import { getKeyFromDate, getWeatherIcon } from 'helpers/weather';
+import { mediaQueryUp } from 'helpers/mediaQueryUp';
+
+import Back from 'icons/Back';
+import { Coordinates } from 'containers/Context/Coordinates';
+import { RouterLinks } from 'components/Routes';
+import TabItem, { tabPrefix } from 'components/TabItem';
+import Title from 'pages/WeatherForecast/components/Title';
+import TitleSmall from 'pages/WeatherForecast/components/TitleSmall';
+import { StyledIcon } from 'containers/StyledIcon';
+
+import * as S from 'pages/WeatherForecast/styled';
+import Loader from 'icons/Loader';
```

Comment:
could you locate this import near other `icons` component, like `import Back from 'icons/Back';`, please?